bodhi/server/validators.py

```diff
@@ -272,9 +272,9 @@ def validate_builds(request, **kwargs):
         up = request.db.query(Update).filter_by(alias=edited).first()
         if not up:
             request.errors.add('body', 'builds',
-                               'Cannot find update to edit: %s' % edited)
+                               'Cannot find update to edit: %s' % request.route_url(edited))
             return
-
+        
```

Comment:
I don't think we need to add a new line here.

Reply:
Let me remove it